### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by lthiery**

### CODE REVIEW

File: lorawan-device/Cargo.toml [hunk 1]
```diff
@@ -56,3 +56,5 @@ external-lora-phy = ["dep:lora-phy"]
 
 ## Enable [`serde`](https://docs.rs/serde/latest/serde/) serialization/deserialization for data structures.
 serde = ["dep:serde", "lorawan/serde"]
+
+default-crypto = ["lorawan/default-crypto"]
```
Comment (plaes) [inline lorawan-device/Cargo.toml:60]: Missing newline at end of line

File: lorawan-device/README.md [hunk 2]
```diff
@@ -3,72 +3,32 @@
...
-
-The following LoRaWAN features are implemented:
+LoRaWAN 1.0.x is approximately implemented:
 
 - Class A device behavior
+- Class C device behavior (async only)
 - Over-the-Air Activation (OTAA) and Activation by Personalization (ABP)
-- Regional support for US915, and EU868
-- Supports CFList in JoinAccept for EU868
+- Regional support for AS923_1, AS923_2, AS923_3, AS923_4, AU915, EU868, EU433, IN865, US915
```
Comment (plaes) [inline lorawan-device/README.md:17]: Probably worth mentioning that region-specific power limits are not yet enforced (see issue #168)

File: lorawan-device/Cargo.toml (hunk 1 above)
Comment (lthiery) [reply lorawan-device/Cargo.toml]: ty

File: lorawan-device/README.md (hunk 2 above)
Comment (lthiery) [reply lorawan-device/README.md]: good idea! thanks

**Review by lucasgranberg — APPROVED** [review]
This is great!

**Pull request merged**